webcompat/static/js/lib/bugform.js

```diff
@@ -428,8 +428,10 @@ function BugForm() {
     copy over the URL of a newly uploaded image asset to the bug
     description textarea.
   */
-  this.addImageURL = function(url) {
-    var imageURL = ['![Screenshot Description](', url, ')'].join('');
+   this.addImageURL = function(url) {
+    var file_ext = url.split('.').pop();
+    var thumb_url = url.replace('.' + file_ext, '-thumb.' + file_ext);
+    var imageURL = ['[![Screenshot Description](', thumb_url, ')](', url, ')'].join('');
```

Comment:
yes that was the purpose of my comment in the pull request. :) I tried to minimize my modification of the JS. And I remember you telling me a couple of months ago that to avoid to repeat the same code, we should probably make a function which is called in the two places. 

I have 0 preferences on the way to do it given my understanding of JS (performance impact, elegance, etc.) The one without Lodash's template seems more direct?